### DISCUSSION TITLE
Test combine?

### DISCUSSION THREAD

**Discussion by filnik** [cashapp/turbine#78] [Q&A] [answered]
Hi, is there a way to test combine of flows with turbine? If yes, can you give me an example on how to do it? Would be great! Thank you

**Comment by JakeWharton** [comment, marked as answer]
The output of combining multiple flows is just a single flow, and Turbine tests single flows.

I would expect an example to look like
```kotlin
combine(flowOf("a"), flowOf("b"), ::Pair).test {
  assertEquals(Pair("a", "b"), awaitItem())
}
```
but I haven't tested it.
